packages/@aws-cdk/aws-codepipeline-actions/test/cloudformation/integ.stacksets.ts

```diff
@@ -0,0 +1,65 @@
+import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import * as s3 from '@aws-cdk/aws-s3';
+import { Asset } from '@aws-cdk/aws-s3-assets';
+import { App, RemovalPolicy, Stack, StackProps } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import * as cpactions from '../../lib';
+
+export class StackSetPipelineStack extends Stack {
+  constructor(scope: Construct, id: string, props: StackProps) {
+    super(scope, id, props);
+
+    const pipeline = new codepipeline.Pipeline(this, 'Pipeline', {
+      artifactBucket: new s3.Bucket(this, 'ArtifactBucket', {
+        removalPolicy: RemovalPolicy.DESTROY,
+      }),
+    });
+
+    const asset = new Asset(this, 'Asset', {
+      path: `${__dirname}/test-artifact`,
+    });
```

Comment:
What's the point of this Asset? Is the idea to actually test the pipeline execution after you deploy this, with the `--no-cleanup` flag?